src/langcheck/metrics/metric_value.py

```diff
@@ -22,6 +22,7 @@ class MetricValue(Generic[NumericType]):
     generated_outputs: List[str]
     reference_outputs: Optional[List[str]]
     sources: Optional[List[str]]
+    explanations: Optional[List[Optional[str]]]
```

Comment:
Why is it `Optional[str]` instead of `str`?

Reply:
Ah I see. How about adding a comment since it's different from the other types?

```suggestion
        explanations: Optional[List[Optional[
            str]]]  # An explanation can be None if the metric could not be computed
```